gap/PolygonalComplexes/graphs.gi

```diff
@@ -296,7 +296,10 @@ if IsPackageMarkedForLoading( "GRAPE", ">=0" ) then
         [IsPolygonalComplex, IsPolygonalComplex],
         function(complex1, complex2)
             local inc1, inc2;
-
+		
+            if IsSimplicialSurface(complex1) and IsSimplicialSurface(complex2) and CounterOfButterflies(complex1)<>CounterOfButterflies(complex2) then
```

Comment:
Why do we check if it is a simplicialsurface as well?

Reply:
The CounterOfButterflies is only defined for simplicial surfaces.

Reply:
If it is not, ```CounterOfButterflies(complex1)<>CounterOfButterflies(complex2)``` would resolve to ```fail<>fail``` so false and it would not be executed anyways. Otherwise one of them is not a simplicial surface and the other one is so they cannot be isomorphic anyways or am I missing something?